app/actions/dat-middleware.js

```diff
@@ -387,19 +387,29 @@ export default class DatMiddleware {
       await mkdirp(this.dataDir)
     } catch (_) {}
 
-    const [ datOpts, paused ] = await Promise.all([
+    const [datOpts, paused] = await Promise.all([
       readJSON(joinPath(this.dataDir, 'dats.json')),
       readJSON(joinPath(this.dataDir, 'paused.json'))
     ])
 
     for (const key of Object.keys(datOpts)) {
-      const opts = JSON.parse(datOpts[key])
-      this.internalAddDat({
-        key,
-        path: opts.dir,
-        paused: paused[key],
-        ...opts
-      })
+      let datdata
```

Comment:
Is there a reason why you renamed the variable?

Reply:
that variable is "not opts". The `opts` have inside of.
below code from `storeOnDisk` method.

```javascript
[key]: {
    dir: this.dats[key].path,
    opts: this.dats[key].opts
}
```